--- ./library/src/tensile_host.cpp	2024-04-19 05:06:26.000000000 +1000
+++ ./library/src/tensile_host.cpp	2025-01-01 19:26:55.582050138 +1000
@@ -690,54 +690,8 @@
                 {
                     base_path = ROCBLAS_LIB_PATH;
 
-                    // Find the location of librocblas.dll/.so
-                    // Fall back on hard-coded path if static library or not found
-
-#ifndef ROCBLAS_STATIC_LIB
-#ifdef WIN32
-                    // wchar_t wpath[MAX_PATH + 1] = {0};
-                    // if(GetModuleFileNameW(GetModuleHandle("rocblas.dll"), wpath, MAX_PATH + 1))
-                    // {
-                    //     std::wstring          wspath(wpath);
-                    //     std::string           tmp(wspath.begin(), wspath.end());
-
-                    std::vector<TCHAR> dll_path(MAX_PATH + 1);
-                    if(GetModuleFileNameA(
-                           GetModuleHandleA("rocblas.dll"), dll_path.data(), MAX_PATH + 1))
-                    {
-                        std::string           tmp(dll_path.begin(), dll_path.end());
-                        std::filesystem::path exepath = tmp;
-                        if(exepath.has_filename())
-                        {
-                            base_path = exepath.remove_filename().string();
-                        }
-                    }
-#else
-                    dl_iterate_phdr(rocblas_dl_iterate_phdr_callback, NULL);
-                    if(rocblas_so_path.size())
-                        base_path = std::string{dirname(&rocblas_so_path[0])};
-#endif
-#endif // ifndef ROCBLAS_STATIC_LIB
-
-                    // Find the location of the libraries
-                    if(TestPath(base_path + "/../../Tensile/library"))
-                        base_path += "/../../Tensile/library";
-                    else if(TestPath(base_path + "/library"))
-                        base_path += "/library";
-                    else if(TestPath(base_path + "/../rocblas/library"))
-                        // For ASAN packaging, library file directory will be lib/asan
-                        // so need to prefix ../ to set search base_path to lib/rocblas/library
-                        base_path += "/../rocblas/library";
-                    else
-                        base_path += "/rocblas/library";
-                }
-                return 0;
-            }();
-
-            path = base_path;
-            if(TestPath(path + "/" + processor))
-                path += "/" + processor;
-
+                    path = "@GENTOO_PORTAGE_EPREFIX@/usr/lib64/rocblas/library";;
+		}
 #ifdef TENSILE_YAML
             tensileLibraryPath = path + "/TensileLibrary_lazy_" + processor + ".yaml";
 #else
